src/templates/PoolManager/Activity/index.tsx

```diff
@@ -224,11 +226,14 @@ const Activity = () => {
       }
     }
 
-    const activitiesInvestors = getActivityInfo(
-      _activities,
-      data.pages[0]?.underlying_assets || [],
-      filters
-    )
+    const assets: Record<string, string> = {}
+    tokenListSwapProvider.forEach(token => {
+      if (token.symbol && token.logoURI) {
+        assets[token?.symbol] = token.logoURI
```

Comment:
If you check if the token exists, you don't need the `?` in here
```suggestion
        assets[token.symbol] = token.logoURI
```